ShippoTesting/TrackTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Shippo;
+
+namespace ShippoTesting
+{
+    [TestFixture ()]
+    public class TrackTest : ShippoTest
+    {
+        [Test ()]
+        public void TestValidGetStatus ()
+        {
+            Shipment shipment = ShipmentTest.getDefaultObject ();
+            Track track = getAPIResource ().RetrieveTracking ("usps", shipment.ObjectId);
+            Assert.IsNotNull (track.TrackingNumber);
+            Assert.IsNotNull (track.TrackingHistory);
+        }
+
+        [Test ()]
+        [ExpectedException (typeof (ShippoException))]
+        public void TestInvalidGetStatus ()
+        {
+            getAPIResource ().RetrieveTracking ("usps", "INVALID_ID");
+        }
+
+        [Test ()]
+        public void TestValidRegisterWebhook ()
+        {
+            Shipment shipment = ShipmentTest.getDefaultObject ();
+            Track track = getAPIResource ().RetrieveTracking ("usps", shipment.ObjectId);
+            Assert.IsNotNull (track.TrackingNumber);
+            Assert.IsNotNull (track.TrackingHistory);
```

Comment:
Asserting these is duplicate effort already done in `TestValidGetStatus`. I think it can be skipped.